### PR TITLE
Dashboard context dropdown position fix on landing page in mobile view.

### PR DESCRIPTION
**Pull request by puni9869**
as title.

Screensots

before 
image

after
image

### CODE REVIEW

**Review by denyskon — APPROVED** [review]

File: web_src/css/dashboard.css [hunk 1]
```diff
@@ -92,6 +92,6 @@
 
 @media (max-width: 767.98px) {
   .dashboard .dashboard-navbar .ui.dropdown > .menu {
-    position: static;
+    position: absolute;
```
Comment (wxiaoguang) [inline web_src/css/dashboard.css]: IIRC the default is "absolute" for menu item.

Comment (puni9869) [reply web_src/css/dashboard.css]: shell i remove this css rule?

Comment (wxiaoguang) [reply web_src/css/dashboard.css]: If it duplicates, why not?

I guess there is some reason for using "static" here, maybe you can try to blame through the history.

Comment (puni9869) [reply web_src/css/dashboard.css]: i will share my findings... on this

Comment (puni9869) [reply web_src/css/dashboard.css]: Actually default pos is absolute but we are using specific `.dashboard .dashboard-navbar ` selector thats why it not picking the default behaviour
Here is the origin of position 

https://github.com/puni9869/gitea/commit/202803fc69d21763a06d8d0f5a4c46509c18f6f1#diff-809c14c56f75fdeee75e624f39fbfa7225c8f8225962aa7e8213944153a75cdaR102

Its not duplicate and was `static` from starting.

Comment (wxiaoguang) [reply web_src/css/dashboard.css]: I believe it's not the root position.

Comment (wxiaoguang) [reply web_src/css/dashboard.css]: The root is here: Fix dashboard switching on Mobile (#20238)

image

Comment (puni9869) [reply web_src/css/dashboard.css]: Thanks for pointing me to the room.

A lot has been change after that in popups and dropdowns. We had a huge fix in mobile view in issue filter dropdowns. We had same problem where we fix the position of element to `absolute`.

At present the current UI is stable in mobile. I did a regression check in all the screens last night.

Comment (wxiaoguang) [reply web_src/css/dashboard.css]: Could we just remove this "absolute" style if the default style is already "absolute"?

Comment (puni9869) [reply web_src/css/dashboard.css]: Working, nicly without absolute too.

**Review by wxiaoguang — APPROVED** [review]

File: web_src/css/dashboard.css (hunk 1 above)
Comment (puni9869) [reply web_src/css/dashboard.css]: image

**Pull request merged**